src/__tests__/index-test.ts

```diff
@@ -45,7 +45,7 @@ describe("AppEnv", () => {
 
   it("can be constructed with missing env vars if skip is set", () => {
     const log = jest.spyOn(global.console, "log").mockImplementation(() => undefined);
-    const invalidEnvVars = { ...validEnvVars };
+    const invalidEnvVars: Partial<typeof validEnvVars> = { ...validEnvVars };
```

Comment:
These are because `delete` complains if the type doesn't mark the deleted property as `optional`:

<img width="642" alt="Screen Shot 2022-02-09 at 17 06 34" src="https://user-images.githubusercontent.com/630449/153311591-0205ada9-2631-49a7-9f9d-db1903a45926.png">